DTM/Balloons/map.json

```diff
@@ -70,7 +70,8 @@
 		}
 	],
 	"itemremove": [
-		"leather helmet", "chainmail chestplate", "leather leggings", "iron boots", "arrow"
+		"stone sword", "bow", "diamond pickaxe", "stone axe", "oak planks", "cooked beef", "arrow", "stone spade",
```

Comment:
"stone spade" isn't the correct item name either, it should just be "stone shovel"